src/main/java/org/folio/inventory/dataimport/consumers/MarcHoldingsRecordHridSetKafkaHandler.java

```diff
@@ -3,7 +3,7 @@
 import static java.lang.String.format;
 
 import static org.apache.commons.lang3.StringUtils.isEmpty;
-import static org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil.constructContext;
+import static org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil.*;
```

Comment:
please use specific imports

Reply:
thank you fixed all, check style didn't inform about it